ui-tests/pom.xml

```diff
@@ -153,11 +153,12 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1798 -->
           <reuseForks>false</reuseForks>
           <environmentVariables>
             <PLUGINS_DIR>../plugin/target/test-classes/test-dependencies</PLUGINS_DIR>
             <JENKINS_VERSION>${jenkins.version}</JENKINS_VERSION>
-            <BROWSER>firefox-container</BROWSER>
+            <BROWSER>chrome-container</BROWSER>
```

Comment:
fwiw - the firefox issue is now fixed in the nightlies
https://bugzilla.mozilla.org/show_bug.cgi?id=1690308